### ISSUE TITLE
RAFT doesn't work with ForkChannels

### ISSUE THREAD

**Issue by belaban** [belaban/jgroups-raft#35]
When creating a `ForkChannel` and adding `RAFT` over it, there's an NPE on `RAFT.init()`. This requires a change in JGroups, see [1] for details. The code below doesn't work

[1] https://issues.jboss.org/browse/JGRP-1926

```java
public class bla {
    protected JChannel   ch, fork_ch;

    protected void start(String name) throws Exception {
        ch=new JChannel("/home/bela/udp.xml").name(name);
        fork_ch=new ForkChannel(ch, "singleton", "fc1",
                                new ELECTION(),
                                new RAFT().members(Arrays.asList("A", "B", "C")).raftId(name),
                                new REDIRECT(),
                                new CLIENT());
        fork_ch.connect("ignored");
        ch.connect("demo");
    }

    public static void main(String[] args) throws Exception {
        new bla().start(args[0]);
    }
}

```

**Comment by belaban** [comment]
Fixed in JGroups 3.6.4; when released we need to update `ivy.xml` and test it.

**Comment by belaban** [comment]
JGRP-1926 contains a new version of bla which uses a `RaftHandle` to modify state. This works with `FORK`. Resolving the issue now.
